core/src/main/java/org/apache/calcite/rex/RexNamedParam.java

```diff
@@ -42,8 +40,7 @@ public class RexNamedParam extends RexVariable {
   public RexNamedParam(
       RelDataType type,
       String name) {
-    super("?" + name, type);
-    this.name = name;
+    super(name, type);
```

Comment:
Changed this because github actions didn't like shadowing the name field/getName functions.